diff --git a/ruvia-web/CMakeLists.txt b/ruvia-web/CMakeLists.txt
index 5f8e936b..85698d26 100644
--- a/ruvia-web/CMakeLists.txt
+++ b/ruvia-web/CMakeLists.txt
@@ -163,10 +163,6 @@ if(RUVIA_ENABLE_POSTGRESQL)
        PostgreSQL_pgport_LIBRARY_DEBUG AND
        PostgreSQL_pgport_LIBRARY_RELEASE)
         target_link_libraries(ruvia-web PUBLIC
-            "$<LINK_GROUP:RESCAN,
-                $<$<CONFIG:Debug>:${PostgreSQL_pgcommon_LIBRARY_DEBUG}>
-                $<$<NOT:$<CONFIG:Debug>>:${PostgreSQL_pgcommon_LIBRARY_RELEASE}>,
-                $<$<CONFIG:Debug>:${PostgreSQL_pgport_LIBRARY_DEBUG}>
-                $<$<NOT:$<CONFIG:Debug>>:${PostgreSQL_pgport_LIBRARY_RELEASE}>>")
+            "$<LINK_GROUP:RESCAN,$<$<CONFIG:Debug>:${PostgreSQL_pgcommon_LIBRARY_DEBUG}>$<$<NOT:$<CONFIG:Debug>>:${PostgreSQL_pgcommon_LIBRARY_RELEASE}>,$<$<CONFIG:Debug>:${PostgreSQL_pgport_LIBRARY_DEBUG}>$<$<NOT:$<CONFIG:Debug>>:${PostgreSQL_pgport_LIBRARY_RELEASE}>>")
     endif()
 endif()
